DESCRIPTION.md

```diff
@@ -11,8 +11,15 @@ Source code is also available at:
 
 - v1.3.5(Unreleased)
```

Comment:
We follow [semantic versioning](https://semver.org/), please proceed accordingly.
I think since you added new feature(s) we should do a minor bump (since previous API was not changed IIRC, but please consider this carefully).

Reply:
1.3.5 if its not a big feature.

Reply:
depending on the fact that there's a bunch of new features, I lean towards to minor bump -> 1.4.0.